lib/src/main/java/com/what3words/androidwrapper/datasource/text/api/dto/LanguageDto.kt

```diff
@@ -0,0 +1,14 @@
+package com.what3words.androidwrapper.datasource.text.api.dto
+
+internal data class LanguageDto(
+    val nativeName: String,
+    val code: String,
+    val name: String,
+    val locales: List<LocaleDto>?
+) {
+    data class LocaleDto(
+        val nativeName: String,
+        val code: String,
+        val name: String
+    )
+}
```

Comment:
There are many fields with the same name which makes me a little bit confused, can you add comments to make it clearer?

Reply:
I deliberately opted not to include comments in the DTOs since they are internal classes solely responsible for parsing responses from the API endpoint. Commenting on one would necessitate adding comments to all, which I believe would be excessive. Therefore, I'll refrain from adding comments to them.